src/main/java/frc/robot/RobotContainer.java

```diff
@@ -176,7 +176,8 @@ private void configureButtonBindings() {
    * @return the command to run in autonomous
    */
   public Command getAutonomousCommand() {
-    return autoChooser.getSelected();
```

Comment:
Is this part of the migration, or is this an out-of-scope change which slipped in?

Reply:
I included this out of scope to test the autonomous command.  I didn't intend for it to be pushed at all, but apparently I didn't clean the working tree before committing.